docs/reference/project/functions.mdx

```diff
@@ -1 +1,46 @@
-TODO...
+A Nitric function is described by its definition in a `nitric.yaml` file. Nitric functions are a type of [Service](../../intro/services) that offers a portable way to develop function-as-a-service (FaaS) applications, which can then be deployed to
+popular Serverless offerings like AWS Lamda, Google CloudRun and Azure AppService.
+
+## Adding a new Function
+
+Start by fetching the latest function templates:
+
+```bash
+nitric templates:repos:add function
+```
+
+Then add the function:
+
+```bash
+nitric make:service
+```
+
+This will populate the `nitric.yaml` file with an entry for the function
```

Comment:
Not currently relevant (this functionality has been removed from the next version of the SDK